app/phonons/libnegfint.F90

```diff
@@ -126,7 +124,6 @@ subroutine negf_init(env, transpar, tundos, initinfo)
       ncont = 0
     endif
 
-    ! ------------------------------------------------------------------------------
     !! Set defaults and fill up the parameter structure with them
```

Comment:
Shouldn't this only use one `!`? And why not also delete the `! ---` lines in 277 and 291?